### ISSUE TITLE
Long nasolabial distance

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6625]
**Preferred term label:** 
Long nasolabial distance

**Synonyms** 

**Definition (free text, please give PubMed ID)** 
Long nasolabial distance on 24459047[PMID]

**Parent term (use hpo.jax.org/app)**
Abnormality of the nasolabial region HP:0005289

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:96164

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think this is the same as Long philtrum, although the authors do not use this phrase (which is preferred).
https://hpo.jax.org/app/browse/term/HP:0000343

Unless you disagree I will add 'Long nasolabial distance' as a synonym to 'Long philtrum'
